server/channels/api4/user.go

```diff
@@ -1539,6 +1539,11 @@ func updateUserActive(c *Context, w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	if user.AuthService == model.UserAuthServiceLdap {
```

Comment:
with this change, when trying to update an ldap user status via mmctl this is the output and thrown error:

```
mmctl user activate 89fmpgso1pytfghejrf5n9ts9o --local
Error: 1 error occurred:
	* unable to change activation status of user: 89fmpgso1pytfghejrf5n9ts9o

```
`{"timestamp":"2024-09-19 11:52:00.490 +02:00","level":"debug","msg":"You cannot modify user status. User is managed by LDAP","caller":"web/context.go:120","path":"/api/v4/users/89fmpgso1pytfghejrf5n9ts9o/active","request_id":"c69hnydfqbrkub17p8hsyyu5dc","ip_addr":"","user_id":"","method":"PUT","err_where":"updateUserActive","http_code":401,"error":"updateUserActive: You cannot modify user status. User is managed by LDAP, userId=89fmpgso1pytfghejrf5n9ts9o"}`

Reply:
That seems like a bug. Only fields that are managed by LDAP should be blocked.

Reply:
oh, by status it refers to user status in the DB (active/inactive) I think it is working as expected.

Reply:
Yes, I read the code in the wrong way.